### DISCUSSION TITLE
Hover formatting question.

### DISCUSSION THREAD

**Discussion by yriveiro** [folke/noice.nvim#715] [Q&A]
Is there any way for the hover functionality to keep the docstring's indentation? As you can see the `files` description is indented but not in hover popup.

```python
"""
    Represents a filesystem with a collection of files, providing methods for accessing
    and manipulating file information.

    Attributes
    ----------
    files : list[str]
        A list of filenames within the filesystem, sorted alphabetically.
    path : pathlib.Path
        A Path object representing the root directory of the filesystem.
   ...
"""
```

<img width="731" alt="Screenshot 2024-02-04 at 11 48 00" src="https://github.com/folke/noice.nvim/assets/934763/b1b066a0-2be5-478c-a6c7-6de240b7882c">

**Comment by Kayzels** [comment]
I've also been looking for something to fix this. If you've found a solution @yriveiro, please let me know.
